dnf5/shared_options.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright Contributors to the dnf5 project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Dnf5 is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "dnf5/shared_options.hpp"
+
+
+namespace dnf5 {
+
+
+void create_allow_downgrade_options(dnf5::Command & command) {
+    auto * repoquery_formatting = command.get_context().get_argument_parser().add_new_group("solver_options");
```

Comment:
It does not affect functionality at all - but still, can you rename this variable (e.g. `solver_options`)?

Reply:
Sure